docs/UserGuide.md

```diff
@@ -406,6 +371,34 @@ If unindicated, the `School Residence` field will default to `DOES NOT LIVE ON C
 * No appointment should clash with any other appointments.
 * The student that the appointment is for must exist in the records.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## FAQ
+
+**Q**: What do I do if I accidentally delete a student? 
+<br>
+**A**: Unfortunately, the deletion is permanent. You will have to manually re-add the student.
+
+**Q**: What can be included in medical details?
+<br>
+**A**: Allergies and past medical history.
+
+**Q** : Will I be able to add other types of appointments besides vaccination appointments?
+<br>
+**A** : No, the current version only allows you to add vaccination appointments and not any other type of appointments. 
```

Comment:
@SiTingST in the addAppt feature section, I wrote that vaccinated students can also have an appointment scheduled for follow-ups. Should I edit my part to say "booster dose" or should the Q&A here say that currently it doesn't differentiate different types of appointments?

Reply:
I think its less confusing if we just remove this question?